### PR TITLE
Add second due date to request form

### PR DESCRIPTION
**Pull request by celestemartinez**
-Add second due date to request form
-Make pdf optional if there is a second due date

### CODE REVIEW

File: openreview/conference/helpers.py [hunk 1]
```diff
@@ -85,6 +94,17 @@ def get_conference_builder(client, request_form_id, support_user='OpenReview.net
     if isinstance(submission_additional_options, str):
         submission_additional_options = json.loads(submission_additional_options.strip())
 
+    if submission_second_due_date:
```
Comment (melisabok) [inline openreview/conference/helpers.py:97]: can we do this inside of the builder?, I guess we already do this.

Comment (melisabok) [reply openreview/conference/helpers.py:97]: here: https://github.com/openreview/openreview-py/blob/master/openreview/conference/builder.py#L1281

if second_due_date exists then content['pdf']['required'] = False

Comment (melisabok) [reply openreview/conference/helpers.py:97]: make sure 'pdf' exists in content

File: openreview/conference/helpers.py [hunk 2]
```diff
@@ -42,15 +42,24 @@ def get_conference_builder(client, request_form_id, support_user='OpenReview.net
         submission_start_date = None
 
     submission_due_date_str = 'TBD'
-    submission_due_date = note.content.get('Submission Deadline', '').strip()
-    if submission_due_date:
+    submission_second_due_date = note.content.get('Submission Deadline', '').strip()
```
Comment (melisabok) [inline openreview/conference/helpers.py:45]: I think it is possible the user enter an abstract deadline but not a submission deadline, what should we do in that case?

**Comment by celestemartinez** [comment]
I was looking at the issue of having `due_date == second_due_date`

I don' think it'll be a problem as https://github.com/openreview/openreview-py/blob/47b6271bc336ab02ba3e9ae5d318bd088e63f9f7/openreview/conference/builder.py#L708 would not be entered and only `setup_final_deadline_stage` would be executed

**Review by melisabok — APPROVED** [review]

**Pull request merged**